connectors/src/connectors/slack/index.ts

```diff
@@ -689,10 +689,7 @@ export class SlackConnectorManager extends BaseConnectorManager<SlackConfigurati
   }
 
   async stop(): Promise<Result<undefined, Error>> {
-    logger.info(
-      { connectorId: this.connectorId },
-      `Stopping Slack connector is a no-op.`
-    );
+    await terminateAllWorkflowsForConnectorId(this.connectorId);
```

Comment:
Should this do the same as pause instead ? i.e also stop incoming webhooks ?

Reply:
The difference between both and the rational behind it 0is not very clear to me - but in all other connectors stop is terminating workflow and not setting pause (causing another monitor to complain about missing workflow). pause is stopping workflows and setting pause. stop can ben called with the cli, but not pause, which is called automatically.

Reply:
Stop / resume was initially just a thing for connectors like notion where you have a long running workflow. Ideally we want to unify at some point. Pause is made to stop running workflows, stop processing webhooks, and stop complaining about the connector in production checks. Here I really think it should either not be implemented or do the same think as Pause

Reply:
For notion it's actually somehow different, as pause does not stop the workflow, only mark the connector as "paused".  Stop does terminate the workflow without setting a pause. If they does the same thing, it would be more clear to have one single action.. but that should be done in a different task, as it implies updating all connectors. Should I keep it "not implemented" for now ?

Reply:
Yes I think not implemented is best. And maybe add pause/unpause to CLI

Reply:
ok, added cli